tests/storage/storage_migration/utils.py

```
@@ -0,0 +1,72 @@
+    assert not rebooted_vms, f"Boot time changed for VMs:\n {rebooted_vms}"
+
+
+def vefiry_vm_storage_class_updated(vm: VirtualMachine, target_storage_class: str) -> None:
+    vm_pvcs_names = [
+        volume["dataVolume"]["name"]
+        for volume in vm.instance.spec.template.spec.volumes
+        if "dataVolume" in dict(volume)
+    ]
+    failed_pvc_storage_check = {}
+    for pvc_name in vm_pvcs_names:
+        pvc_storage_class = PersistentVolumeClaim(namespace=vm.namespace, name=pvc_name).instance.spec.storageClassName
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)